### DISCUSSION TITLE
AUTH_FAILED/Your credentials might be wrong

### DISCUSSION THREAD

**Discussion by denimolsen** [qdm12/gluetun#1351] [Q&A]
I am using PIA for my VPN service and in my logs I am getting an "AUTH_FAILED". "Your credentials might be wrong." And I am sure that I am inputting my username and password for PIA. Anyone have experience with this? Everything else is great, but the container can't even run properly because the auth is failing. Any help is appreicated.

**Comment by bnhf** [comment]
You might need to put your username and/or password in quotes if they contain any "special characters". Otherwise, have you tried logging into the same server group using PIA's desktop client to be sure that works?
